Modules/GameState.cs

```diff
@@ -621,6 +621,12 @@ public void Update(PlayerControl player)
 
             }
 
+            if (player.Is(CustomRoles.Tired) && player.IsAlive())
+            {            
+                Tired.SendRPC(player.PlayerId);
```

Comment:
Rpc is sent but never received? If you aren't receiving the rpc then there is no point in sending it.

Also, if you do send it, it should be below `Tired.AfterActionTasks(player);` because that is where the playerIdList js being changed